presto-testing-kafka/src/main/java/io/prestosql/testing/kafka/TestingKafka.java

```diff
@@ -34,7 +34,7 @@
 
     public TestingKafka()
     {
-        container = new KafkaContainer("5.4.1");
+        container = new KafkaContainer("5.4.1").withNetworkAliases("kafka");
```

Comment:
Unrelated change?

Reply:
I had it in a separate commit but squashed it after previous review.
Should I put it back in a separate commit?

Reply:
Yes, since it's a logically independent change, it's better to track it separately.